lib/ssh_scan/update.rb

```diff
@@ -5,6 +5,10 @@
 
 module SSHScan
   class Update
+    def initialize
+      @errors = []
+    end
```

Comment:
I'm using this in bin/ssh_scan, so the user sees an error message whenever an exception occurs (say we cannot connect to rubygems or something failed).